src/main/java/core/basesyntax/service/impl/ReaderServiceCsvImpl.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ReaderService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ReaderServiceCsvImpl implements ReaderService {

```

Comment:
This service responsible for reading data from file and parsind it to FruitTransaction, this violates SRP, lets create separate services for this